tasks/10_checks-repos.yml

```diff
@@ -67,13 +67,16 @@
 
     - name: Install icoutils in local controller
       ansible.builtin.package:
-        name: "{{ alternc_icoutils_pkg }}"
+        name: '{{ alternc_icoutils_pkg }}'
```

Comment:
Why this change from double quotes to single? Am I wrong or did you change something that works and interpolates the variable to force a string with double mustaches on it?

Reply:
For esthetic and harmonization reasons, I try to use single quotes as first. this came from a proposed and merged branch, so I harmonize my way. 

Is there a difference between " and ' here?¡

Reply:
Improving esthetic and harmonization are more than valid reasons, and are appreciated.

> Is there a difference between " and ' here?

No, there is no difference. At first I thought that using single quotes would not interpret and interpolate the variable `alternc_icoutils_pkg` in the `name` parameter, but I was wrong. Momentary lapse that a local test quickly cleared up.